### ISSUE TITLE
PhysX Force Region point force requires much greater magnitude value than other forces to trigger effects on entities

### ISSUE THREAD

**Issue by LB-MateuszSzweda** [o3de/o3de#11707]
**Describe the bug**
**PhysX Force Region** component _Magnitude_ value needed to move regular sized entities has to be about 1000 times greater when the component force is set to _Point_ force than when set to other forces such as _Local Space_ or _World Space_ .

For instance moving a box shaped Rigid body entity with X:1.Y:1,Z:1 dimensions will require setting **PhysX Force Region** component Magnitude value to _100000.0_ when _Point_ force is used, whereas _Local Space_ force will require Magnitude value of 100 to trigger similar effect on the entity.

PhysX Debug is especially helpful with observing this behavior.

Please note that this issue also occurs on Linux platform and on the Development (9459742) builds.

For further information please refer to the attached video file.

**Prerequisites**

**PhysX Debug** Gem is enabled and built for the active project.

**Steps to reproduce**

1. Launch the Editor and create a new level.
2. Create an entity with **PhysX Collider** and **PhysX Force Region** components. 
3. Set the **PhysX Collider** component shape to _Box_ and set X:5.Y:5,Z:5 as its dimensions.
4. Turn on the _Trigger_ toggle in the **PhysX Collider** and _Debug Forces_ in **PhysX Force Region** component.
5. In the **PhysX Force Region** component set its  force type to _Point_ with the _Magnitude_ value of  _100.0_.
6. Duplicate the entity and set its **PhysX Force Region** force type to _Local Space_ with the _Magnitude_ value of  _100.0_.
7. Place the duplicated entity next to the first entity.
8. Create a new entity with **PhysX Collider** and **PhysX Rigid Body** components.
9. Set the **PhysX Collider** component shape to _Box_ and set X:1.Y:1,Z:1 as its dimensions.
10. Enable PhysX debug by typing _Physx_Debug 1_ command in the console.
11. Position the Rigid body entity above the first Force Region entity and enter the game mode.
12. Position the Rigid body entity above the second Force Region entity and enter the game mode.

**Expected behavior**
Both _Point_ and _Local Space_ forces are able to move the rigid body entity when the _Magnitude_ value  is set to _100.0_.

**Actual behavior**
 _Point_ Force Type requires _Magnitude_ value  of _100000.0_ to move the entity.

**Screenshots/Video**

https://user-images.githubusercontent.com/86953456/187916999-995805ae-90a8-4214-ba90-76e7b036749a.mp4

**Found in Branch**
Stabilization/2210 (d3760da)

**Found in Branch**
Development (52ebc81)

**Desktop/Device:**
- Device: PC
- OS: Windows
- Version: 10 Pro
- CPU: AMD Ryzen 9 3900X
- GPU: NVIDIA GeForce RTX 2060 SUPER
- Memory: 16GB

**Comment by hultonha** [comment]
This seems like a good first issue to follow the repro steps and replicate the problem and try and debug and see what is happening. Please feel free to tag myself @hultonha or @greerdv for suggestions.

Note: This may be related to mass (noted by @greerdv)

**Comment by greerdv** [comment]
Yeah, the point force doesn't scale with mass but the local space force does, and the entity that's falling into the force region is going to be pretty heavy (not sure off the top of my head what the default density is but with a 1x1x1 box collider I guess the mass is going to end up in the thousands of kg).

It's not super clear to me what the correct behaviour should be with regards to scaling with mass, so I think we need to put some thought into that.

**Comment by Riju-bak** [comment]
Hi @LB-MateuszSzweda , @hultonha and @greerdv. I see that this issue is still open. If no one else is working on it, can I give it a try?

**Comment by LB-MateuszSzweda** [comment]
@Riju-bak I confirm that the issue is still open and active on both Windows and Linux on latest Development (b4ca369) however I don't know if any developer is working on it or is assigned to it. 

I'll be able to verify the fix as a QA if the fix will be committed to the Development branch.

**Comment by Riju-bak** [comment]
Oh alright, thanks for letting me know @LB-MateuszSzweda. I'll try it out on my local machine. Seems like an interesting exercise.

**Comment by Riju-bak** [comment]
Okay, so according to step 4 in the instructions for [Building for Windows](https://docs.o3de.org/docs/welcome-guide/setup/setup-from-github/building-windows/), 

`This step is optional because in the “source engine” build model, the engine is built inside of every project. If you plan on working with projects, to avoid building the engine twice, consider waiting until you learn how to create and build a project`

It seems to me that to replicate this issue, I need to create a new project. But if I make any changes to engine source, how will that be reflected the next time I open Editor.exe from within my project's build folder?

I apologize if this question is dumb, but the 4th point has confused me.

**Comment by LB-MateuszSzweda** [comment]
@Riju-bak It all depends on the workflow but generally speaking all changes to the code provided to source engine will require rebuilding the created project to update it with provided changes, which does not take a lot of time because only the changed code will be detected as modified and rebuilt.


The step 4 is intended to spare the user some time because creating project via CLI with for instance already existing default template prior to configuring/building engine will indeed be faster than building the source engine and then using the Project Configurator to create and build the project.

So in short to have the shortest possible walkthrough through source configuration the user would.

1. Clone the engine and create new project with CLI by following https://docs.o3de.org/docs/user-guide/project-config/cli-reference/ and for instance using
 `o3de.bat create-project --project-path PROJECT_PATH --template-path C:\o3de\Templates\DefaultProject` 
 command to create it.
2. Register the project
3. Configure and build the engine pointing to created project so:

`cmake -B build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_PROJECTS=(Created Project name)`

`cmake --build build/windows_vs2019 --target  Editor --config profile -- -m`

4. Commiting changes to code and rebuilding the project (this can be done with project configurator then).

But as the issue can be reproduced on any Source/Pre-built SDK  build types and any project I would suggest doing something I do whenever I am more focused on the Engine rather than the project so using AutomatedTesting project.

This project is already present in the cloned o3de repository and is generally configured with every basic functionality so spares the time and hassle of creating a new project.

In this case the steps are as simple as

1. Clone the o3de repository.
2. Obtain python packages with get_python.sh script
3. Configure and build the engine with

`cmake -B build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_PROJECTS=AutomatedTesting`
`cmake --build build/windows_vs2019 --target Editor --config profile -- -m`

4. Commit changes to the code, rebuild AutomatedTesting and test the changes.

**Comment by Riju-bak** [comment]
Thanks a lot @LB-MateuszSzweda .  I'll try this out in a couple of days
